Makefile

```diff
@@ -41,11 +41,6 @@ clean:
 dependencies: carthage-bootstrap configs secrets fabric
 
 bootstrap: hooks dependencies
-	brew update || brew update
-	brew unlink swiftlint || true
-	brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/686375d8bc672a439ca9fcf27794a394239b3ee6/Formula/swiftlint.rb
```

Comment:
If we're removing the pinning of `swiftlint` here do we have the binary checked into the repo?

Reply:
Good question...I'll make it so